packages/esm-service-queues-app/src/queue-table/queue-table.test.tsx

```diff
@@ -96,8 +95,8 @@ describe('QueueTable', () => {
       const patientName = entry.patient.person.display;
       const row = screen.getByText(patientName).closest('tr');
 
-      expect(within(row).getByText(entry.status.display)).toBeInTheDocument();
-      expect(within(row).getByText(entry.priority.display)).toBeInTheDocument();
+      expect(within(row!).getByText(entry.status.display)).toBeInTheDocument();
+      expect(within(row!).getByText(entry.priority.display)).toBeInTheDocument();
```

Comment:
I don't think we'll need this non-null assertion check here once we make the changes I've suggested above.